### ISSUE TITLE
Hard to locate searched node in map

### ISSUE THREAD

**Issue by Garbash** [willzeng/WikiNets#121]
I can think of several sophisticated options to solve this but the simplest one would be to choose a brighter color for the selected node :)

**Comment by zhangjiannan** [comment]
I would suggest to highlight the node and highlight the connected links.
As in Graphpedia: http://www.jiannanweb.com/graphpedia/index.php

**Comment by Garbash** [comment]
It's a cool concept.
When I click on a resource, I see on the list view the resource and it's
connections, so why not highlight the connections as well?
This logic works for selecting a connection - It should highlight the
connection and the two or more resources it connects.

On Thu, Feb 20, 2014 at 12:29 AM, zhangjiannan notifications@github.comwrote:

> I would suggest to highlight the node and highlight the connected links.
> As in Graphpedia: http://www.jiannanweb.com/graphpedia/index.php
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/willzeng/WikiNets/issues/121#issuecomment-35557834
> .

## 

Get some insight of what makes me tick:
My twitter https://twitter.com/#!/garbash
Rhizi.net mailing list https://groups.google.com/forum/#!forum/rhizinet

Dor Garbash
108 Rue du faubourg du temple, Paris 75011
+33-652919878

**Comment by zhangjiannan** [comment]
That is totally true! Good thing to implement in WikiNets, we can send a feature request.

**Comment by Garbash** [comment]
Done :)
https://github.com/willzeng/WikiNets/issues/122

On Thu, Feb 20, 2014 at 5:11 PM, zhangjiannan notifications@github.comwrote:

> That is totally true! Good thing to implement in WikiNets, we can send a
> feature request.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/willzeng/WikiNets/issues/121#issuecomment-35630766
> .

## 

Get some insight of what makes me tick:
My twitter https://twitter.com/#!/garbash
Rhizi.net mailing list https://groups.google.com/forum/#!forum/rhizinet

Dor Garbash
108 Rue du faubourg du temple, Paris 75011
+33-652919878
